apps/thehive/README.md

```diff
@@ -0,0 +1,5 @@
+# Vast - TheHive integration
+
+Vast is perfect companion for Security specialists, threat hunters, SOC analysts who are utilizing TheHive. Using Vast via multiple interfaces (analyzer, webhook, etc) investigations can be easily enriched and contextualized.
+
+Example `docker-compose.yml` can be customized to fit any requirements. After instantiating environment, manual integration between TheHive and Cortex should be done.
```

Comment:
In general, we strive to write documentation by using active voice and present tense. This makes technical documentation actionable and easier to read and understand. Lead with the problem, and then follow up with the solution.

I'll make a pass over the wording in a final review, so I've not commented on this any further. It'll take a while to get used to the writing style for technical documentation (at least it took me a few months to do so when I joined), but with a little practice I can see you getting there very quickly. 🙏

Reply:
I used the same approach on official TheHive analyzer repo, where I believe our analyzer will also end up in. https://github.com/TheHive-Project/Cortex-Analyzers

we can change the documentation based on feedback of course.

Reply:
I made changes on this document and used active voice. Kindly let me know if further changes are needed.